libsolidity/parsing/Token.h

```diff
@@ -403,5 +311,29 @@ class Token
 	static char const m_tokenType[NUM_TOKENS];
 };
 
+class ElementaryTypeNameToken
+{
+public:
+	ElementaryTypeNameToken(Token::Value token, std::string const& description)
```

Comment:
`detail` instead of `description`? And please add `_` prefixes to all parameter names.